src/sk_transformers/number_transformer.py

```diff
@@ -109,9 +109,7 @@ def transform(self, X: pd.DataFrame) -> pd.DataFrame:
                 if value is None:
                     X[new_column] = op(X[feature], **kwargs or {})
                 elif isinstance(value, str):
-                    X[new_column_with_value] = op(
-                        [X[feature], X[value]], **kwargs or {}
-                    )
+                    X[new_column_with_value] = op(X[feature], X[value], **kwargs or {})
```

Comment:
Hm, now `np.prod` is not working anymore. 🤔 It would be great if we find a solution for it too.

Reply:
One should use `np.multiply`.